multicluster/build/yamls/antrea-multicluster.yml

```diff
@@ -1553,56 +1564,20 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
         volumeMounts:
-        - mountPath: /tmp/k8s-webhook-server/serving-certs
-          name: cert
-          readOnly: true
         - mountPath: /controller_manager_config.yaml
           name: manager-config
           subPath: controller_manager_config.yaml
-      securityContext:
-        runAsNonRoot: true
       serviceAccountName: antrea-multicluster-controller
       terminationGracePeriodSeconds: 10
       volumes:
-      - name: cert
-        secret:
-          defaultMode: 420
-          secretName: webhook-server-cert
       - configMap:
           name: antrea-multicluster-manager-config
         name: manager-config
 ---
-apiVersion: cert-manager.io/v1
-kind: Certificate
-metadata:
-  labels:
-    app: antrea
-  name: antrea-multicluster-serving-cert
-  namespace: kube-system
-spec:
-  dnsNames:
-  - antrea-multicluster-webhook-service.kube-system.svc
-  - antrea-multicluster-webhook-service.kube-system.svc.cluster.local
-  issuerRef:
-    kind: Issuer
-    name: antrea-multicluster-selfsigned-issuer
-  secretName: webhook-server-cert
----
-apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
-  labels:
-    app: antrea
-  name: antrea-multicluster-selfsigned-issuer
-  namespace: kube-system
-spec:
-  selfSigned: {}
----
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  annotations:
-    cert-manager.io/inject-ca-from: kube-system/antrea-multicluster-serving-cert
+  creationTimestamp: null
```

Comment:
this line is added by `make manifests` automatically, let me check if there is configuration to skip it.

Reply:
hi @quan, looks like it's a known bug https://github.com/kubernetes-sigs/controller-tools/issues/402, I didn't see a fix in k8s upstream, since we will add auto-generation script and new structure of manifests for both member and leader cluster, so I will leave this to another PR after @aravindakidambi 's common area and manifests auto-generation script fix.